src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,42 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+type TodoInfoProps = {
+  todo: {
+    id: number,
+    title: string,
+    completed: boolean,
+    userId: number,
+    user: {
+      id: number,
+      name: string,
+      username: string,
+      email: string,
+    }
+  }
```

Comment:
```suggestion
  todo: Todo

```
just import the Todo type from the type file